packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -712,6 +725,8 @@ class FlameChartNode<T> {
   final Color backgroundColor;
   final Color textColor;
   final T data;
+  final bool Function(T) useAlternateBackground;
```

Comment:
this should probably be a method rather than a Function.

Reply:
discussed offline. Added a related todo to FlameChartNode class